ex04.ex

```diff
@@ -55,7 +55,15 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(list) do
```

Comment:
# 5

although I'd like to see the "min" part broken out into its own function.